style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+#title {
+  font-size: 1.8rem;
+  font-family: sans-serif;
+  margin: 4rem auto;
+  text-align: center;
+}
+
+body {
+
+  background-color: greenyellow;
+  color: rgb(39, 36, 36);
+  width: 100%;
+  height: auto;
+}
+
+#survey-form {
+  padding: 3rem 2.5rem;
+  border-radius: 7px;
+}
+
+#submit {
+  background-color: aqua;
+  color: white;
+  padding: 0.4rem;
+  cursor: pointer;
+  font-size: 1.15rem;
+}
```

Comment:
Please include linters for easy debugging

Reply:
Great job being polite.

Reply:
Great job being polite.